src/components/LayoutTree.tsx

```diff
@@ -551,6 +608,21 @@ const TreeNode = ({
               prevWidth={treeThoughtsPositioned[index - 1]?.width}
             />
           )}
+        <span
+          className={css({
+            color: 'blue',
```

Comment:
You can add a new color definition (`caret`? `selection`?) to https://github.com/cybersemics/em/blob/main/src/colors.config.ts.